File: boot-web/src/main/java/com/hjf/boot/demo/web/Man.java
package com.hjf.boot.demo.web;

/**
 * name: com.hjf.boot.demo.web
 * author: mickjoust
 * date: 2018/4/25
 **/
public class Man {

    private String name;
    private Integer phone;

    public Man() {
    }

    public Man(String name, Integer phone) {
        this.name = name;
        this.phone = phone;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getPhone() {
        return phone;
    }

    public void setPhone(Integer phone) {
        this.phone = phone;
    }
}

File: boot-cache/src/main/kotlin/com/hjf/boot/demo/boot/cache/kotlin/Man.kt
package com.hjf.boot.demo.boot.cache.kotlin

//import javax.persistence.*

/**
 * name: com.hjf.boot.demo.boot.high_concurrency.kotlin
 * author: mickjoust
 * date: 2018/5/9
 **/
//@Entity
//@Table(name="users")
class Man(
//        @Id @GeneratedValue(strategy = GenerationType.AUTO)
        var id: Int = -1,
        var name: String = "",
        var email: String = ""
) {
    override fun toString(): String {
        return "Man(id=$id, name='$name', email='$email')"
    }
}